src/StackExchange.Redis/Hacks.cs

```diff
@@ -1,5 +1,7 @@
-﻿#if !NET5_0_OR_GREATER
-
+﻿#if NET5_0_OR_GREATER
```

Comment:
Should it be `NET6_0_OR_GREATER` since we currently support `net6.0` other than `net5.0`

Reply:
The "currently" is the key word there. Ok, in reality we're not going to add a 5.0 at this point, but by sticking with the relevant framework version *for the feature*, if we change the target frameworks: any such `#if` will sort themselves out automatically, without anything needing to be revisited.

In short: I'm happier with the `NET5_OR_GREATER` here

Reply:
Agreed - updating all the definitions each time is precisely what these defines we're meant to avoid. It tells us which version this code started to be needed in and that's not a terrible thing in itself. I'd vote not to chase these overall - it's just a lot of yearly diff that never ends :)